crates/cheatnet/src/forking/data.rs

```diff
@@ -0,0 +1,38 @@
+use crate::runtime_extensions::forge_runtime_extension::contracts_data::build_name_selector_map;
+use starknet::core::types::ContractClass;
+use starknet::core::types::contract::AbiEntry;
+use starknet_api::core::{ClassHash, EntryPointSelector};
+use std::collections::HashMap;
+
+#[derive(Default)]
+pub struct ForkData {
+    pub abi: HashMap<ClassHash, Vec<AbiEntry>>,
```

Comment:
nit: Could we rename it so indicates plural? `abis` seems weird (I assume you may have considered it), so maybe something else

Reply:
Oh, right 😅